components/Profile/EditProfileModal.tsx

```diff
@@ -85,7 +96,7 @@ const EditProfile = ({
                 backdrop={closable ? true : "static"}
                 keyboard={false}
                 centered={true}
-                size="md"
+                size="lg"
```

Comment:
Any specific reason for this?

Reply:
`Type '"md"' is not assignable to type '"sm" | "lg" | "xl"'.ts(2322)`
`(JSX attribute) ModalProps.size?: "lg" | "sm" | "xl"`

I changed the value due to this error.

Reply:
Okay, no problem them.